documentation/docs/pages/operators/nodes/nym-node/setup.mdx

```diff
@@ -0,0 +1,179 @@
+
+## Current version
+
+```
+<!-- cmdrun ../../../../target/release/nym-node --version -->
+```
+
+## Summary
+
+<VarInfo/ >
+
+To run a new node, you can simply execute the `nym-node` command without any flags. By default, the node will set necessary configurations. If you later decide to change a setting, you can use the `-w` flag.
+
+The most crucial aspect of running the node is specifying the `--mode`. At the moment it can be only one of three: `mixnode`, `entry-gateway`, and `exit-gateway`.
+
+Currently the `nym-node` binary can only be run in a single `--mode` at any one time. In the future however, operators will be able to specify multiple modes that a single `nym-node` binary can run. Our goal is to have as many nodes as possible enabling multiple modes, and allow the Nym API to position the node according the network's needs in the beginning of each epoch.
+
+Every `exit-gateway` mode is basically an `entry-gateway` with NR (Network Requester) and IPR (IP Packet Router) enabled. This means that every `exit-gateway` is automatically seen as an `entry-gateway` but not the opposite.
+
+Gateway operators can check out the node performance, connectivity and much more in our new tool [harbourmaster.nymtech.net](https://harbourmaster.nymtech.net/).
+
+To determine which mode your node is running, you can check the `:8080/api/v1/roles` endpoint. For example:
+```sh
+# sustitude <IPv4_ADDRESS> or <HOSTNAME> with the one corresponding to your node
+# for http
+http://<IPv4_ADDRESS>:8080/api/v1/roles
+# or
+http://<IPv4_ADDRESS>/api/v1/roles
+
+# for reversed proxy/WSS
+https://<HOSTNAME>/api/v1/roles
+```
+
+Everything necessary will exist on your node by default. For instance, if you're running a mixnode, you'll find that a NR (Network Requester) and IPR (IP Packet Router) address exist, but they will be ignored in `mixnode` mode.
+
+For more information about available endpoints and their status, you can refer to:
+```sh
+# sustitude <IPv4_ADDRESS> or <HOSTNAME> with the one corresponding to your node
+# for http
+http://<IPv4_ADDRESS>:8080/api/v1/swagger/#/
+# or
+http://<IPv4_ADDRESS>/api/v1/swagger/#/
+
+# for reversed proxy/WSS
+https://<HOSTNAME>/api/v1/swagger/#/
+```
+
+## Usage
+
+### Help Command
+
+There are a few changes from the individual binaries used in the past. For example by default `run` command does `init` function as well, local node `--id` will be set by default unless specified otherwise etcetera.
+
+<Callout color="info" emoji="ℹ️">
+You can always use `--help` flag to see the commands or arguments associated with a given command.
+</Callout>
+
+Run `./nym-node --help` to see all available commands:
+
+```
+<!-- cmdrun ../../../../target/release/nym-node --help -->
+```
+
+To list all available flags for each command, run `./nym-node <COMMAND> --help` for example `./nym-node run --help`:
+
+```
```

Comment:
ditto

Reply:
I left all cmdrun for easier location post merge